client/src/components/row/Row.js

```diff
@@ -23,30 +25,50 @@ function rowDeletionMode(props) {
                 </div>
                 <div className="rightRow">
                     <div className="proposal-title">{props.title}</div>
-                    <img src={props.vote} alt='Vote Status' className="vote-status"></img>
+                    <img src={yesIcon} alt='Vote Status' className="vote-status"></img>
                 </div>
             </div>
         </div>
     );
 }
 
 function rowDashboardMode(props) {
-    return (
-        <div
-            className="proposal-box"
-            id={props.title}
-            onClick={() => {props.changeTitle(props.title);
-                            props.changeDescription(props.description);
-                            props.changeWantedPropID(props.id);
-                            props.changeTextBoxValue()}
-        }>
-        
-            <div className="proposal-title">{props.title}</div>
-            <img src={props.vote} alt='Vote Status' className="vote-status"></img>
-
-        </div>
-        
-    );
+    const boolean = props.voteStatus;
+    if (boolean) {
+        return (
+            <div
+                className="proposal-box"
+                id={props.title}
+                onClick={() => {props.changeTitle(props.title);
+                                props.changeDescription(props.description);
+                                props.changeWantedPropID(props.id);
+                                props.changeTextBoxValue()}
+            }>
+            
+                <div className="proposal-title">{props.title}</div>
+                <img src={yesIcon} alt='Vote Status' className="vote-status"></img>
+    
+            </div>
+            
+        );
+    } else {
+        return (
+            <div
+                className="proposal-box"
+                id={props.title}
+                onClick={() => {props.changeTitle(props.title);
+                                props.changeDescription(props.description);
+                                props.changeWantedPropID(props.id);
+                                props.changeTextBoxValue()}
+            }>
+            
+                <div className="proposal-title">{props.title}</div>
+                <img src={noIcon} alt='Vote Status' className="vote-status"></img>
+    
+            </div>
+            
+        );
+    }
```

Comment:
Not critical, but you could make this condition simpler by only putting the condition in `<img src={boolean ? yesIcon : noIcon} alt='Vote Status' className="vote-status"></img>`